eng/pipelines/coreclr/pr.yml

```diff
@@ -30,176 +12,111 @@ jobs:
 - template: /eng/pipelines/common/checkout-job.yml
 
 #
-# Debug builds
-#
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
-    buildConfig: debug
-    platforms:
-    - Windows_NT_x64
-    - Windows_NT_x86
-    jobParameters:
-      testGroup: innerloop
-
-#
-# Checked builds
+# Build CoreCLR
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
     jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
     buildConfig: checked
     platforms:
-    - Linux_arm
-    - Linux_arm64
-    - Linux_musl_x64
-    - Linux_x64
     - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
-    - Windows_NT_x64
+    - Linux_x64
     - Windows_NT_x86
-    jobParameters:
-      testGroup: innerloop
-
-#
-# Release builds
-#
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
-    buildConfig: release
-    platforms:
-    - Linux_arm64
-    - Linux_musl_x64
-    - Linux_rhel6_x64
-    - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
     - Windows_NT_x64
+    - Windows_NT_arm64
     jobParameters:
       testGroup: innerloop
 
 #
-# Checked test builds
+# Build CoreFX using live CoreLib from CoreCLR
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-test-job.yml
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
     buildConfig: checked
     platforms:
-    - Linux_arm
-    - Linux_arm64
     - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
-    - Windows_NT_x64
+    - Linux_x64
     - Windows_NT_x86
+    - Windows_NT_x64
+    - Windows_NT_arm64
     jobParameters:
-      testGroup: innerloop
+      isOfficialBuild: false
+      testScope: empty
+      useLiveCoreClr: true
 
 #
-# Checked JIT test executions
+# Test builds using live CoreFX
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/run-test-job.yml
+    jobTemplate: /eng/pipelines/coreclr/templates/build-test-job.yml
     buildConfig: checked
     platforms:
-    - Linux_arm
-    - Linux_arm64
-    - Linux_musl_x64
-    - Linux_x64
     - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
-    - Windows_NT_x64
+    - Linux_x64
     - Windows_NT_x86
-    helixQueueGroup: pr
-    helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+    - Windows_NT_x64
+    - Windows_NT_arm64
```

Comment:
Tests aren't arch specific, so you should be able to just build `Windows_NT_x64`.